test/unit/rules/lint-img-alt-attributes-test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'img-alt-attributes',
+
+  config: true,
+
+  good: [
+    '<img alt="hullo">',
+    '<img src="hello.jpg" alt="">'
```

Comment:
Personally, I'd prefer to require at least one char in `alt`.  Which would make this actually a `bad` example.

Sorry to be so wishy washy on it...